### DISCUSSION TITLE
Twitch : Category search function problem

### DISCUSSION THREAD

**Discussion by Alzyon** [WarmUpTill/SceneSwitcher#942] [General]
Hi,

Since the last update, when I want to add a Twitch category for a game, the search function doesn't find any games.

Thank you for your help.

**Comment by WarmUpTill** [comment]
Hi,
just to clarify:
The functionality was previously working with 1.24.0 and now you can no longer find any Twitch categories with version 1.24.1?
If so, that is very strange - I don't think there were any changes in that area.

Which category are you trying to search for / what are your search terms?
Does it work again if you delete the connection to your Twitch account and request a new token?

**Reply by Alzyon** [reply]
When I ask for a new token on the Macro, I get to the Twitch page which asks me once again for "Advanced Switcher" authorisations, but when I click on "Authorise", it loads and then an error appears "Connection failed".

In fact, the version I was using of this plugin was the beta "1.24.0-beta1", which included Twitch categories for the 1st time. I've now noticed that the name change is no longer automatic on all my games, whereas it was working a month ago.

Could this be a Twitch problem ?

**EDIT** : I completely erased the macro and recreated it. Then I deactivated "Ublock" on my browser for this page, and apparently it seems to be working again, because I no longer have the error and I've been able to search the categories again.

The problem was obviously on my side.

Thank you for your reply. ^^

**Reply by WarmUpTill** [reply]
Ah, great!
Thanks for the update! :)
